jina/__init__.py

```diff
@@ -11,6 +11,7 @@
 
 import platform
 import sys
+from typing import Dict
```

Comment:
we are very restrictive on the import in `__init__.py` because it will slow down the starting speed of Jina CLI, e.g. `jina -v`

Reply:
please remove it

Reply:
Thanks for your response.
Remove the import in a fixup commit (will rebase and force push later).